.serena/memories/code_style_and_conventions.md

```
@@ -0,0 +1,74 @@
+# Code Style & Conventions — Omega_KG
+
+## Python Style
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)